code/modules/mob/living/simple_mob/subtypes/animal/pets/dog.dm

```diff
@@ -244,5 +242,27 @@
 	icon_living = "brittany"
 	icon_dead = "brittany_dead"
 
+/mob/living/simple_mob/animal/passive/dog/german
+	name = "german shepherd"
+	real_name = "german shepherd"
+	desc = "It's a german sherpherd."
```

Comment:
What's a german *sherpherd*?

Reply:
Obviously a misspelling of `german shampoodle`

Reply:
resolved